tests/test_dataset_manifest.py

```diff
@@ -7,6 +7,9 @@
 from collections import Counter
 from unittest.mock import patch
 
+from PIL import Image
```

Comment:
need test in test_manifest_dataset.py as well

Reply:
Could you remind me which test we should add to the test_manifest_dataset.py for the image matting data?

Reply:
create_matting_dataset?